scripts/ci/install-rust.sh

```diff
@@ -2,8 +2,17 @@
 
 set -ex
 
+git clone https://github.com/influxdata/flux.git flux-repo
+cd flux-repo
+git fetch --tags
+latest_tag=$(git describe --abbrev=0 --tags)
+git checkout $latest_tag -b latest-tag
```

Comment:
Building off your comment above, I agree that always pulling the latest Flux tag probably isn't the right thing to do. If we wanted to keep a 1:1 match with Flux then I'd expect us to inspect the `go.mod` to find the tag/commit of Flux that we're actually depending on, and check out that. We could use `go list` to extract the version information, then parse the output to extract the tag-or-commit to check out.

IMO it might be more complicated than it's worth. My preference would be to keep the Rust version pinned as a constant here.

Reply:
There's a desire from product and other members of Edge to follow the rust version in Flux for master/2.0

Reply:
Looked into a `go list` way we could do it, and I think this will work. Assuming we've checked out `influxdb` and are running from the project root (which we currently do in CI), this command will print the path to the copy of `flux` downloaded by Go:
```
go list -m -f '{{ .Dir }}' github.com/influxdata/flux
```
It will work regardless of whether we currently depend on a "proper" version of flux vs. a snapshot build. We can `cat` the flux CI config out of it, something like this:
```
FLUX_DIR="$(go list -m -f '{{ .Dir }}' github.com/influxdata/flux)"
RUST_LATEST_VERSION=$(cat "${flux_dir}/.circleci/config" | grep RUST_VERSION: | sed 's/^ *//' | cut -d ' ' -f2)
```

Reply:
ok, looks like a few minor tweaks will be required but I'll make the change